### PR TITLE
Added element data module

### PR DESCRIPTION
**Pull request by YHordijk**
Element data module contains colors and radii of all atoms with an easy to use interface.

### CODE REVIEW

File: TCutility/atom_data.py
Comment (SiebeLeDe) [inline TCutility/atom_data.py:1]: Please, try to rewrite the code with:
import pathlib as pl

and then us pl.Path to use join paths etc. Pathplib is sincerely recommended to use!

**Review by SiebeLeDe — COMMENTED** [review]
See the comment about rewriting atom_data.py with pathlib instead of using os

File: TCutility/atom_data.py
Comment (YHordijk) [reply TCutility/atom_data.py:1]: Done!

**Review by SiebeLeDe — COMMENTED** [review]
Nice use of pathlib!

**Pull request merged**